src/app/donation-start/donation-start-form/donation-start-form.component.scss

```diff
@@ -516,3 +516,9 @@ mat-expansion-panel-header h3 {
   margin-top: 30px;
   position: absolute;
 }
+
+div.frc-captcha {
+  margin: auto;
+  background: none;
+  border: 2px solid $colour-primary;
+}
```

Comment:
this is slightly different to the one on the login page, on that I didn't apply any stiles so it follows the default white background with no border. I think no background and blue border looks better though, so I could change that too - probably will add the change to this PR.